conf/config.yaml

```diff
@@ -43,5 +43,5 @@
 apisix:
   admin_key:
     - name: admin
-      key: edd1c9f034335f136f87ad84b625c8f1  # using fixed API token has security risk, please update it when you deploy to production environment
+      key: ${{APISIX_ADMIN_API_KEY}}
```

Comment:
According to the discussion on the mailing list, the default value should be empty? I don't understand what the environment variables here mean

Reply:
do we need to update https://github.com/apache/apisix/blob/master/conf/config-default.yaml#L91-L100?

Reply:
any do need to remove https://github.com/apache/apisix/blob/master/apisix/admin/init.lua#L64-L68?

Reply:
I think it would be more flexible to replace with an environment variable. Users can modify the admin token through system environment variables. If the environment variable is not read. At this point, APISIX will generate a random token for it and prompt it on standard output and logs.